prep-html-skeleton/index.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE>
+<html>
+
+<head>
+  <title>HTML Skeleton Excercise</title>
+</head>
+<body>
+  <h1>I built a proper HTML skeleton!</h1>
+</body></html>
```

Comment:
You should not put two closing tags on the same line like this, because it can be easy to miss if you are trying to read through your code later. Please move `</html>` to its own line